Fixing the header and background

audreytoskin opened the pull request:
Solves Issue #62.

audreytoskin committed:
Strip trailing whitespace.

My editor kept doing this automatically, and it doesn't hurt anything.

audreytoskin committed:
Remove background images, let elements fall naturally on page.

The positioning of the background images in the header and footer were a
recurring problem (and anyway they kinda looked like stock images for a
WordPress theme). I also don't think the footer is important enough that
it needs to *always* be on screen, if the user's display is small enough
to require scrolling. So, this way makes it easier to avoid having the
footer image appearing too high, or having various elements overlap each
other.

terrycloth commented:
Basically, _this_ is what it looks like now.

![screen shot 2017-09-05 at 20 07 02-fullpage](https://user-images.githubusercontent.com/3881885/30094075-7d953c66-927e-11e7-9c35-a4aaac9fc663.png)

terrycloth commented:
Maybe don't merge this just yet, though. I'm noticing some weird things still happening in:

* Epiphany (desktop Webkit browser) --- does not adjust layout at all for smaller display sizes.
* Firefox and Chrome on Android --- does not show the bigger logo (or _any_ image) for landscape layout.
* Dolphin (an Android browser) --- footer is bunched up.

audreytoskin committed:
Switch to @media queries based on pixels instead of rem.

Some web clients were rendering the page inconsistently around the
"small screens & mobile" display size, because there was some overlap
between the first two @media queries: a display size that is
*approximately* 40rem wide would satisfy both the max-width and
min-width queries. Also, the media queries for medium and large screens
*change the default font size*, which made calculating the display width
in rem extra weird.😅

I also made the break point between the "small" and "medium" display
sizes a little narrower than the equivalent rem had been, so that my
phone shows the wider grid of buttons, requiring less scrolling, in
landscape layout.

terrycloth commented:
I think it should be good now.

terrycloth commented:
I'm of course open to feedback 🙂

terrycloth commented:
Whatcha think?
